web/services/HolidayService.php

```diff
@@ -188,6 +188,7 @@ static function groupByWeeks(array $leavesDetails, $weeks = []): array
                 $weeks[$current_week] = $leavesDetails[$dates[$i]]['amount'] ?? 1;
                 $previous_week = $current_week;
             }
+            $weeks[$current_week] = round($weeks[$current_week], 2);
```

Comment:
Sorry, I cannot reproduce a situation that generates a report with long decimal numbers. We have already rounded partial leaves in `mapHalfLeaves`, which is used as input of this function. Isn't that enough?

Reply:
I thought it was enough too but I think summing them in the previous for loop generates unexpected results. I could reproduce it by manually adding a holiday of 1:33hour

![Screen Shot 2022-06-08 at 13 56 33](https://user-images.githubusercontent.com/333447/172610506-38e38808-18aa-490d-be15-ce8cdfec74e8.png)

Then you can see it in the Vacation Management
![Screen Shot 2022-06-08 at 13 57 53](https://user-images.githubusercontent.com/333447/172610722-4e868df3-b598-446f-bd8c-1bb11e3a7d94.png)

and Vacation Summary:
![Screen Shot 2022-06-08 at 13 57 43](https://user-images.githubusercontent.com/333447/172610725-0c5ac736-0bec-438c-919c-010fba4c4dce.png)